lib/election.js

```diff
@@ -855,6 +883,10 @@ function _getManifestFromPeer(
         return _validateEvents(
           ledgerNode, manifest.item, manifest.blockHeight, callback);
       }
+
+      // FIXME: bypassing this check, votes are in the process of being saved
```

Comment:
@dlongley I'd like to get this PR merged, and then create another issue for re-implementing the vote check here.